lib/modal-box/index.d.ts

```diff
@@ -1,96 +1,97 @@
-declare module Coveo.ModalBox {
+declare namespace Coveo.ModalBox {
```

Comment:
I tried to revert this change along with the tabbing, but it looks like whenever I commit my changes, the file gets re-formatted like this. I'll write comments under the lines I changed in this file.